app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -178,9 +193,12 @@ public Result deleteSurvey(String surveyGuid, String createdOnString) throws Exc
         Survey survey = surveyService.getSurvey(keys);
         verifySurveyIsInStudy(session, studyId, survey);
         
-        surveyService.deleteSurvey(survey);
+        if ("true".equals(physical) && user.isInRole(Roles.ADMIN)) {
+            surveyService.deleteSurveyPermanently(survey);
```

Comment:
Unit tests? For both deleting permanently and for if they try to call this without admin privileges?

Do we want to throw Unauthorized if they aren't an admin? Or do we want to fall back to logical delete?